src/index.js

```diff
@@ -24,4 +118,8 @@ app.listen(port, err => {
     return;
   }
   console.log(`Listening http://localhost:${port}`);
-});
\ No newline at end of file
+});
+
+function randomNum () {
+  return  Math.floor(Math.random() * 1001);
```

Comment:
1001 no me dice nada, podría ser una variable que explique que es:

```suggestion
  let TOTAL_ITEMS = 1001;
  return  Math.floor(Math.random() * TOTAL_ITEMS);
```

Reply:
Ok, anotado para futuros desarrollos